### DISCUSSION TITLE
Brotherhood cards

### DISCUSSION THREAD

**Discussion by Soul-Gravier** [secretcowlevel/DoomtrooperPublic#56] [Card Balance]
Considering I've been playing Brotherhood only and exclusively for weeks, I'd like to have my say on some of the faction cards and suggest some changes.
Warrior:
- Nikodemus: I know it will be boosted, so I wait before giving my impressions.
- Creshaw: it's ok but I expect more from a rare character. I only play it because it's like having (almost) a fourth Majoris. Honestly prefer to pay him 1 V more with S 6 or with the ability of the Mortificators.
Support:
Nothing to declare.
Attachment:
- Purifier: it's really bad. As I said earlier it would be enough to give it + 5 S and Burn to make it playable. But it would still be worse than the Justifier.
- Mortis Sword: I expect more from a rare attachment. You don't play many Arts and the ones you have you want to use. you don't want to put them on the sword for a miserable +2 C.
the justifier is stronger also in this case. I would add First Strike to make it at least playable.
Art:
- Banishment: I don't like it. It's always worse than Hasty Gateway.
For two reasons:
1. I can play it in combat and in the opponent's turn instead of mine at the cost of two actions.
2. Very often the strongest warriors have powerimmunity and it doesn't work.
I would change it to make it playable but be careful not to make it too strong.
- Dispose: 3 actions and 15 DP… Unplayable. Also for the usual speech of many strong warriors with powerimmunity. I would halve the DP, even at the cost of making it Restricted.
- Exorcise Injuries: Why should I use this and not First Aid Kit? It does not make sense. It is really useless.
I would do it like this: Art Persistent with 2 "heal" charges.
- Domination / Explosion / Gusts of Esitation: As long as there are 3 Reverse of Fortune in each deck they will be unplayable.

**Comment by TomekPL-1** [comment]
I use Domination and Gust of Hesitation in one of my Brotherhood deck. I do not agree that is "unplayable"

**Reply by Kite80** [reply]
Explosion has a small cost compared to what it does, especially when caster has slayer and first strike or invincible.

Exorcise injuries is far better than first aid kit, considering that you can get it back with Cathedral.

Brotherhood can have up to 5 actions per turn thanks to 4th directorate and book of law, so cards which require 2 or more actions to be played can be used without too much effort.

**Reply by 21-ric** [reply]
Archinquisitor Nikodemus : https://github.com/secretcowlevel/DoomtrooperPublic/discussions/2

Crenshaw the redeemer : he's not the best warrior but he is a good SLAYER.

AC-41 Purifier : https://github.com/secretcowlevel/DoomtrooperPublic/discussions/37

Mortis Sword : I'm not a fan of this BLADE but it could help for discard cards. 
For example, you could discard "mental shield" against a player who doesn't play POWER cards.
:arrow_right: FIRSTSTRIKE is a good idea for this card.

Armor Of The True Assassin : https://github.com/secretcowlevel/DoomtrooperPublic/discussions/24

Mask Of The Vestals : This card is not yet available and we started a discussion. https://github.com/secretcowlevel/DoomtrooperPublic/discussions/18
EQUIPMENT, UNIQUE, FACTION
Brotherhood recipient only. Whenever this Warrior's controller plays a card from his or her hand, the player may immediately pay 1 [PP] to draw a card to replace it. Discarded cards are not replaced in this manner.

We need to be careful with POWER cards. Some cards in database will help their use.

Mystic (BROTHERHOOD) [C]
When another Warrior is casting an ART spell, you may discard this Warrior to provide +10 [DP] to the spell effects. Assume the caster is spending this [DP] .
F: 4 S: 4 A: 4 V: 5   

Brother Claudius (BROTHERHOOD) [U]
UNIQUE, MORTIFICATOR
During COMBAT, every 1 [DP] Claudius spent on ART spells he casts counts as 3 [DP] .
F: 7 S: 7 A: 6 V: 8

Nicholai (BROTHERHOOD) [R]
UNIQUE, SLAYER
Mortificator. Every 1 [DP] Nicholai spends on Personal COMBAT Spells counts as 2 [DP] .
F: 5 S: 4 A: 6 V: 8

Exorcist Glove (BROTHERHOOD) [U]
EQUIPMENT, UNIQUE, FACTION
BROTHERHOOD recipient only. Every 1 [DP] spent on this caster's spells counts as 2 [DP] . Warrior gains [F] +3 and [S] +3 when battling DARKLEGION Warriors, and an additional to [F] and [S] when battling HERETICs.
